### ISSUE TITLE
Presentation

### ISSUE THREAD

**Issue by OskarHavo** [OskarHavo/M7011E-Labb#2]
TL;DR - 5-10 minutes showcasing and discussion, in English; You'll have to do it to pass, but it won't have huge impact on your grading.

    10 minutes per group, divided as 5 minutes presentation and 5 minutes Q/A
    Should contain
        A pitch of your Application (Sinek style, or the like)
        Design decisions
            System Sketch (napkins please)
            API
            Component implementations (choices & why)
        Tour of the application in action (or stills)
    Only 5 minutes padding between groups, and before first presentation; so come sharp!

Whilst presentation is mandatory for passing the course, details of content will be evaluated in your report for grading, not on your execution of the presentation; you'll have to do it, but it won't typically influence your grading.
